src/OP.ts

```diff
@@ -117,6 +147,23 @@ export class OP {
     };
   }
 
+  public async selectVerifiableCredentialsForSubmission(
```

Comment:
I think we might want to move this method to the agent as well. Then the flow becomes to use the peAgent if VCs are being used. Do the selection and validation over there. The OP class just has the definitation and accepts VPs

Reply:
I've added this method here because I've considered it a method for our interface.
app (or whatever)  that uses this lib will call this method first and based on the result creates the correct presentation_submission object and sends the correct VCs